dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.connectors-2.0.feature

```diff
@@ -0,0 +1,7 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jakarta.connectors-2.0
+singleton=true
+-bundles=io.openliberty.jakarta.connectors.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.resource:jakarta.resource-api:2.0.0-RC1"
+kind=beta
```

Comment:
Please make this `noship` for your initial delivery. Even if you could go straight to beta, the private feature below is noship and that won't work.